packages/validate/stories/Overview.stories.mdx

```diff
@@ -52,11 +52,11 @@ class MyValidator extends Validator {
   }
 
   execute(modelValue, param) {
```

Comment:
Same here, I think

```js
execute() {
  return modelValue === param;
}
```

Is clear enough.

Reply:
It should be clear the execute function will show any feedback if it returns true. The way I understood it at first is "when validator returns false, there is an error". This should be clarified imo. The example shouldn't be written efficient I think. It should just explain how it work. What do you think?

Reply:
For me an example should showcase usage. 

Documentation / types (jsdocs in our case) should annotate what the return value is and what that means in my opinion.

I get your point of view now though and maybe it is indeed nice that the example provides clarity on top of docs. this PR is an improvement for sure. I'm fine with going with your changes ^^ I will edit my review